spec/recipes/minimize_access_spec.rb

```diff
@@ -27,56 +27,62 @@
     stub_command("find /bin  -perm -go+w -type f | wc -l | egrep '^0$'").and_return(false)
   end
 
-  let(:chef_run) do
+  cached(:chef_run) do
     ChefSpec::ServerRunner.new.converge(described_recipe)
   end
 
+  subject { chef_run }
+
   it 'remove write permission from /usr/local/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/sbin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/sbin'
+    ).with(
       command: 'chmod go-w -R /usr/local/sbin'
     )
   end
 
   it 'remove write permission from /usr/local/bin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/bin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/bin'
+    ).with(
       command: 'chmod go-w -R /usr/local/bin'
     )
   end
 
   it 'remove write permission from /usr/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/sbin').with(
+    is_expected.to run_execute('remove write permission from /usr/sbin').with(
```

Comment:
[The default is 80 characters](https://github.com/bbatsov/rubocop/blob/v0.47.1/config/default.yml#L1198), which is what I recommend as well; 80 character lines display nicely on GitHub without the need for scrolling. Switching over to [Cookstyle](https://github.com/chef/cookstyle) also seems like a good way to align cookbook style specifically with the Chef community.